integration_tests/integration/summary.spec.ts

```diff
@@ -102,6 +102,36 @@ context('Summary', () => {
     })
   })
 
+  it('Risk visible on page', () => {
+    const summaryPage = Page.verifyOnPage(SummaryPage)
+    summaryPage.riskTitle().should('have.text', 'Risk')
+    cy.get('#risk .govuk-summary-list').getSummaryList().should('deep.equal', {
```

Comment:
can `.get('#risk .govuk-summary-list')` be pushed into the SummaryPage into a function?

Reply:
ignore me, see its been done that way with other sections